webapp/sources/rudder/rudder-web/src/main/webapp/secure/configurationManager/directiveManagement.html

```diff
@@ -33,6 +33,25 @@ <h1>Directives</h1>
         </div>
       </div>
       <div data-lift="configuration.DirectiveManagement.userLibrary" ></div>
+      <script>
+        var main = document.getElementById("filters-app")
+        var initValues = {
+            contextPath : contextPath,
+            objectType  : "directive"
+        };
+        filterApp = Elm.Filters.init({node: main, flags: initValues});
+        filterApp.ports.toggleTree.subscribe(function(result) {
+          toggleTree('#activeTechniquesTree', this);
+        });
+        filterApp.ports.searchTree.subscribe(function(filters) {
+          $("#activeTechniquesTree").jstree("searchtag", filters.filter, filters.tags, {"key":false , "value":false});
+        });
+        filterApp.ports.sendFilterTags.subscribe(function(tags) {
+          if (typeof tagsApp === "undefined") return false;
+          tagsApp.ports.getFilterTags.send(tags);
+        });
+        $(".bsTooltip").bsTooltip({container: "#boxDirectiveTree" })
+      </script>
```

Comment:
The script was within the `configuration.DirectiveManagement.userLibrary` template which updates (destroys) itself after a directive is created, so we also need to move it up here